linux/devdoc/error_handling_requirements.md

```diff
@@ -0,0 +1,41 @@
+# error handling
+
+## Overview
+
+Linux implentaton of GetLastError and SetLastError 
+
+## Exposed API
+
+```c
+uint64_t last_error_code;
+
+MOCKABLE_FUNCTION(, void, error_handling_set_last_error, uint32_t, err_code);
+MOCKABLE_FUNCTION(, uint32_t, error_handling_get_last_error);
+```
+
+### set_last_error
+
+```c
+MOCKABLE_FUNCTION(, void, error_handling_set_last_error, uint32_t, err_code);
+```
+`error_handling_set_last_error` implements the PAL wrapper for the SetLastError function from Windows.
+
+`error_handling_set_last_error` shall succeed and assign a non-`NULL` value to `last_error_code`.
```

Comment:
>succeed and [](http://example.com/codeflow?start=38&length=11)

Remove the words "succeed and".  Since this returns void there is no succeeding or failing, it just does some work.